quotes.json

```
@@ -4826,7 +4825,8 @@
   },
   {
     "author": "Peter Drucker",
-    "text": "The best way to predict your future is to create it."
+    "text": "You cannot predict the future, but you can create it.",
+    "source": "https://books.google.com/books?id=mV3XM28AdoIC&dq=%22You+cannot+predict+the+future,+but+you+can+create+it"
```

Restore deep link to quote in book:
```suggestion
    "source": "https://books.google.de/books?id=mV3XM28AdoIC&pg=PA4&hl=en&sa=X&#v=onepage&q=%22You%20cannot%20predict%20the%20future%2C%20but%20you%20can%20create%20it.%22"
```